docs/kubernetes/how-to-get-a-cluster-running.md

```diff
@@ -58,6 +73,27 @@ Create a cluster on your server and define IP range for pods.
 kubeadm init --pod-network-cidr=192.168.0.0/16
 ```
 
+Configure kubectl
+```bash
+mkdir -p ~/.kube
+sudo cp -i /etc/kubernetes/admin.conf ~/.kube/config
+sudo chown $(id -u):$(id -g) ~/.kube/config
+```
+
+Install network addon ([list of addons](https://kubernetes.io/docs/concepts/cluster-administration/addons/))
```

Comment:
I was confused by the formulation a little bit - I would rather say something like
"Install [Calico](https://docs.projectcalico.org/v3.3/getting-started/kubernetes/) network [add-on](https://kubernetes.io/docs/concepts/cluster-administration/addons/)."

Reply:
The meaning is that you can choose any of the listed add-ons, but I have added an example from Calico, that we use.

Reply:
Ok, so I would rewrite it in this way then: "Install a network add-on. You can choose anything from [the list](https://kubernetes.io/docs/concepts/cluster-administration/addons/). For purpose of this guide, we will install [Calico](https://docs.projectcalico.org/v3.3/getting-started/kubernetes/)".

I do  not want to be a nitpick, however, I was confused when I first read it so I wat to improve the formulation :slightly_smiling_face:

Reply:
fixed :slightly_smiling_face: